packages/client/src/pages/Login.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export const Login = () => {
+  const onSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+  };
+
+  return (
+    <form onSubmit={onSubmit}>
+      <label htmlFor="username">Username or Email</label>
+      <input
+        name="username"
+        id="username"
+        type="text"
+        onChange={(e) => {
+          // todo
+        }}
+      />
+
+      <label htmlFor="password">Password</label>
+      <input
+        name="password"
+        id="password"
+        type="password"
+        onChange={(e) => {
+          // todo
```

Comment:
do you think at this time we need to include the forgot password option here or should we proceed ahead without it atm?

Reply:
I think we can just leave it, we'll need to do it for front end integration anyway